### DISCUSSION TITLE
How to center text  in markdown and export it to pdf

### DISCUSSION THREAD

**Discussion by cofeek-codes** [jupyterlab/jupyterlab#16687] [Q&A] [answered]
I know this might be stupid, but i can't solve it

For example i have this document
```md
---
    title: Test document
    author: cofeek-codes
    date: 17/08/2024
---

# Some header

<center>Centered text</center>

``` 
and i am exporting it to pdf using `pandoc doc.md -o doc.pdf` but in pdf this text is not centered

**Comment by ranjiGT** [comment, marked as answer]
This happens because Pandoc doesn’t support the `<center>` HTML tag when converting to PDF (via LaTeX). It gets ignored during the conversion.

---

### ✅ Correct way to center text in Pandoc

Use raw LaTeX instead:

```markdown
\begin{center}
Centered text
\end{center}
```

---

### ✅ Alternative (Pandoc Markdown)

You can also use:

```markdown
::: center
Centered text
```

**Reply by cofeek-codes** [reply]
Time passed, but thanks for solution anyway)
